platform/security/sts/security-sts-guestvalidator/src/main/java/org/codice/ddf/security/validator/guest/GuestValidator.java

```diff
@@ -65,10 +57,11 @@ private GuestAuthenticationToken getGuestTokenFromTarget(ReceivedToken validateT
     }
 
     private boolean validIpAddress(String address) {
-        return IPV4_PATTERN.matcher(address)
-                .matches() || IPV6_STD_PATTERN.matcher(address)
-                .matches() || IPV6_HEX_COMPRESSED_PATTERN.matcher(address)
-                .matches();
+        try {
+            return InetAddress.getByName(address) != null;
+        } catch (UnknownHostException e) {
```

Comment:
@stustison may know better, but I can't think of any *reasonable* chance. I don't find any usages of `java.lang.SecurityManager` in our codebase. Since there's no real expectation of it being thrown, I think it is appropriate not to swallow an unchecked exception.

That's my reasoning, but I could be persuaded otherwise. :)